src/renderer/windows/settings/advanced-settings.tsx

```diff
@@ -22,6 +25,23 @@ export const AdvancedSettings: FunctionComponent = () => {
         <Row>
           <StyledCheckbox color={INPUT_COLOR} label="Enable dev tools" size="xs" {...register('isDebug')} />
         </Row>
+        <Row>
+          <Label>
+            Refresh Time (milliseconds)
```

Comment:
Don't use milliseconds here, use seconds instead

```suggestion
            Refresh Time (seconds)
```